### ISSUE TITLE
Hi

### ISSUE THREAD

**Issue by NikNikYkt** [ouldevloper/Laravel_telegram_bot#1]
What is it? It telegram bot? How to use, install, run?

**Comment by ouldevloper** [comment]
yeah it s a telegram bot, you havent to install any thing , just run add your bot token to .env file and run 
php artisan  serve 
on the project directory
